src/shared/components/BaseToastMessages.tsx

```diff
@@ -0,0 +1,153 @@
+import {
```

Comment:
Since this is the first shared UI component that involves Tailwind, we'll need to make some changes to the Tailwind config to ensure they generate styles for components in this module. I suggest you ping Lyza and ask for her input on how to do this.